src/packages/user/user/repository/user-repository-base.ts

```diff
@@ -0,0 +1,32 @@
+import { UMB_USER_STORE_CONTEXT_TOKEN, UmbUserStore } from './user.store.js';
+import { UMB_USER_ITEM_STORE_CONTEXT_TOKEN, UmbUserItemStore } from './item/user-item.store.js';
+import type { UmbControllerHostElement } from '@umbraco-cms/backoffice/controller-api';
+import { UmbContextConsumerController } from '@umbraco-cms/backoffice/context-api';
+import { UMB_NOTIFICATION_CONTEXT_TOKEN, UmbNotificationContext } from '@umbraco-cms/backoffice/notification';
+
+export class UmbUserRepositoryBase {
```

Comment:
I'm not sure if we have made Repositories a Controller anywhere jet. But I would say that would be the right for this one as well.. So `new UmbContextConsumerController(this.host, ` can be changed into `this.consumeContext(...`

Reply:
I have made a RepositoryBase now that extends the BaseController. The code for all user repositories has been updated. Now we just have to go through all the repos. I think we will have to do that in the next PR.